omod/src/main/java/org/openmrs/module/reportingrest/web/resource/ReportDesignResource.java

```diff
@@ -0,0 +1,111 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
```

Comment:
This is the current license header: https://github.com/openmrs/openmrs-core/blob/2.6.4/api/src/main/java/org/openmrs/Address.java#L1-L9